src/components/ResultsSorter/ResultsSorter.tsx

```diff
@@ -4,48 +4,28 @@ import './ResultsSorter.scss';
 
 interface Props {
     moviesCount: number;
-}
-
-interface State {
     sortBy: string;
+    onSortByChange: Function;
 }
 
-export default class extends React.PureComponent<Props, State> {
-    public constructor(props: Props) {
-        super(props);
-
-        this.state = {
-            sortBy: 'rating'
-        };
-
-        this._handleSortByClick = this._handleSortByClick.bind(this);
-    }
-
-    private _handleSortByClick(value: string) {
-        this.setState({
-            sortBy: value
-        });
-    }
-
-    public render() {
-        return (
-            <div className="results-sorter">
-                <div>{`${this.props.moviesCount} movie(s) found`}</div>
-                <div>
-                    <span>Sort by</span>
-                    <SortByButton
-                        value="release date"
-                        selected={this.state.sortBy === 'release date'}
-                        handleSortByClick={this._handleSortByClick}
-                    />
-
-                    <SortByButton
-                        value="rating"
-                        selected={this.state.sortBy === 'rating'}
-                        handleSortByClick={this._handleSortByClick}
-                    />
-                </div>
+export default React.memo((props: Props) => (
+    <div className="results-sorter">
+        <div className={!props.moviesCount ? 'hidden' : ''}>
+            <div>{`${props.moviesCount} movie(s) found`}</div>
```

Comment:
That can be simplified into
```
<div>{props.moviesCount} movie(s) found</div>
```
Or even less verbose with props destructing:
```
<div>{moviesCount} movie(s) found</div>
```

Reply:
react/jsx-one-expression-per-line

to avoid warnings I must write:
```
<div>
    {props.moviesCount}
    movie(s) found
</div>
```
it looks extreamly strange for me

Reply:
The latter is also will remove space before `movie(s)`, but I got your point.